src/main/resources/com/google/api/codegen/ruby/apiary_names_update.sh

```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This script is no longer relevant; it was previously used to generate a Ruby names file from our baseline discovery docs to allow our generator tests to pass. (The process logic came directly from the generate script in the client library.) But we've now copied over the full names file from the updated client library, which covers all the APIs.